pkg/kubelet/dockertools/manager.go

```diff
@@ -473,6 +473,65 @@ func (dm *DockerManager) GetPodInfraContainer(pod kubecontainer.Pod) (kubecontai
 	return kubecontainer.Container{}, fmt.Errorf("unable to find pod infra container for pod %v", pod.ID)
 }
 
+// makeEnvList converts the map to a list of strings, in the form of
+// '<key>=<value>', which can be understood by docker.
+func makeEnvList(envs map[string]string) (result []string) {
+	for k, v := range envs {
+		result = append(result, k, v)
```

Comment:
You're right @vmarmol re append, but it doesn't matter if we revert to
using the list.

On Wed, May 13, 2015 at 12:17 PM, Victor Marmol <notifications@github.com>
wrote:

> In pkg/kubelet/dockertools/manager.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/8146#discussion_r30248014>
> :
>
> > @@ -473,6 +473,65 @@ func (dm *DockerManager) GetPodInfraContainer(pod kubecontainer.Pod) (kubecontai
> >  	return kubecontainer.Container{}, fmt.Errorf("unable to find pod infra container for pod %v", pod.ID)
> >  }
> >
> > +// makeEnvList converts the map to a list of strings, in the form of
> > +// '<key>=<value>', which can be understood by docker.
> > +func makeEnvList(envs map[string]string) (result []string) {
> > +	for k, v := range envs {
> > +		result = append(result, k, v)
>
> I don't think this inserts with an "=" in between? But we may remove this
> if we change to a list.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/8146/files#r30248014>
> .
>